Description: Fix version number
Author: Léo Haf <leo@haf.ovh>
Forwarded: not-needed
Last-Update: 2025-07-24
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.13)
 
-project(DATUM VERSION 0.3.2 LANGUAGES C)
+project(DATUM VERSION 0.4.0 LANGUAGES C)
 
 # Enable C23 if supported, else fall back to C11 for compatibility
 if(CMAKE_VERSION VERSION_LESS "3.21")
--- a/src/datum_protocol.h
+++ b/src/datum_protocol.h
@@ -45,7 +45,7 @@
 // Works out to over 5 minutes of jobs at 30-40 second work change intervals. No miner should be holding on to work this long.
 #define MAX_DATUM_PROTOCOL_JOBS 8
 
-#define DATUM_PROTOCOL_VERSION "v0.3.2-beta" // this is sent to the server as a UA
+#define DATUM_PROTOCOL_VERSION "v0.4.0-beta" // this is sent to the server as a UA
 #define DATUM_PROTOCOL_CONNECT_TIMEOUT 30
 
 #define DATUM_PROTOCOL_MAX_CMD_DATA_SIZE 4194304 // 2^22 - protocol limit!
